src/jdk.jlink/share/classes/jdk/tools/jlink/resources/jlink.properties

```diff
@@ -111,10 +111,16 @@ main.extended.help.footer=\
 \              used, one pattern per line\n\
 \n\
 
+main.capability.runtime=Capabilities: {0}run-time-image
 
 error.prefix=Error:
 warn.prefix=Warning:
 
+err.runtime.link.not.linkable.runtime=The current run-time image does not support run-time linking.
```

Comment:
should we use "runtime" instead of "run-time" in the help message and errors/warnings?   "runtime" is already used in the messages in this file.

Reply:
Prefer “run-time” when it’s an adjective (e.g., “run-time image”); prefer “runtime” when it’s a noun (e.g., “the Java runtime”).

Reply:
I've tried to use `run-time image` and `runtime` as suggested throughout now.